static/src/stylesheets/module/experiments/_acquisitions-epic-testimonials.scss

```diff
@@ -0,0 +1,89 @@
+// CSS used for the Epic testimonials test
+
+// Common classes
+
+.epic__testimonial__container {
+    margin-top: $gs-gutter;
+    margin-bottom: $gs-gutter;
+}
+
+.epic__testimonial__quote {
+    position: relative;
+}
+
+.epic__testimonial__quote span {
+    position: absolute;
+}
+
+.epic__testimonial__quote svg {
+    fill: #ffbb00;
```

Comment:
Please don't introduce explicit colors, but use one of [the variables](http://guardian.github.io/pasteup-palette/demo/): This could be `$multimedia-main-2`.